Wrappers/Python/cil/framework/acquisition_geometry.py

```diff
@@ -2146,6 +2159,9 @@ def set_angles(self, angles, initial_angle=0, angle_unit='degree'):
         :return: returns a configured AcquisitionGeometry object
         :rtype: AcquisitionGeometry
         '''
+        if AcquisitionType.CONE_FLEX & self.geom_type:
+            raise NotImplementedError("Angles cannot be set for Cone3D_Flex geometry.")
```

Comment:
Shouldn't be not implemented 
should be a warnings.warn "Angles cannot be set for Cone3D_Flex geometry.", then it will do nothing

Reply:
also update unit test